README.md

```diff
@@ -162,35 +195,102 @@ Once this is done, there are two ways to run the operator:
 NAME                     DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
 memcached-operator       1         1         1            1           1m
 ```
 
+Verify that the memcached-operator pod is up and running:
+
+```sh
+$ kubectl get pod
+NAME                                  READY     STATUS    RESTARTS   AGE
+memcached-operator-7d76948766-nrcp7   1/1       Running   0          44s
+```
+
+**IMPORTANT:** Ensure that you built and pushed the image, and updated the `operator.yaml` file.   
+
+Verify that the operator is running successfully by checking it logs. 
+
+```sh
+$ kubectl logs memcached-operator-7d76948766-nrcp7
+{"level":"info","ts":1567613603.7161574,"logger":"cmd","msg":"Go Version: go1.12.7"}
+{"level":"info","ts":1567613603.7163043,"logger":"cmd","msg":"Go OS/Arch: linux/amd64"}
+{"level":"info","ts":1567613603.7163143,"logger":"cmd","msg":"Version of operator-sdk: v0.10.0+git"}
+{"level":"info","ts":1567613603.7166178,"logger":"leader","msg":"Trying to become the leader."}
+{"level":"info","ts":1567613603.8369129,"logger":"leader","msg":"No pre-existing lock was found."}
+{"level":"info","ts":1567613603.8667152,"logger":"leader","msg":"Became the leader."}
+{"level":"info","ts":1567613603.9393756,"logger":"cmd","msg":"Registering Components."}
+{"level":"info","ts":1567613603.9396026,"logger":"kubebuilder.controller","msg":"Starting EventSource","controller":"memcached-controller","source":"kind source: /, Kind="}
+{"level":"info","ts":1567613603.9398162,"logger":"kubebuilder.controller","msg":"Starting EventSource","controller":"memcached-controller","source":"kind source: /, Kind="}
+{"level":"info","ts":1567613604.0514739,"logger":"metrics","msg":"Metrics Service object created","Service.Name":"memcached-operator-metrics","Service.Namespace":"myproject"}
+{"level":"info","ts":1567613604.0976534,"logger":"cmd","msg":"Could not create ServiceMonitor object","error":"no ServiceMonitor registered with the API"}
+{"level":"info","ts":1567613604.0986068,"logger":"cmd","msg":"Install prometheus-operator in your cluster to create ServiceMonitor objects","error":"no ServiceMonitor registered with the API"}
+{"level":"info","ts":1567613604.0988326,"logger":"cmd","msg":"Starting the Cmd."}
+{"level":"info","ts":1567613604.2993534,"logger":"kubebuilder.controller","msg":"Starting Controller","controller":"memcached-controller"}
+{"level":"info","ts":1567613604.3995395,"logger":"kubebuilder.controller","msg":"Starting workers","controller":"memcached-controller","worker count":1}
+```
+
+The following error will be faced if your cluster was unable to pull the image. 
+
+**NOTE:** Just for tests purposes make the image public and setting up the cluster to allow use insecure registry. ( E.g --insecure-registry 172.30.0.0/16 )  
+
+```sh
+$ kubectl get pod
+NAME                                  READY     STATUS             RESTARTS   AGE
+memcached-operator-6b5dc697fb-t62cv   0/1       ImagePullBackOff   0          2m
```

Comment:
I would move this entire block just below the "The follo9wing error will occur ..." BEFORE the **NOTE:** line. 

It should read as follows:

The following error will occur if your cluster was unable to pull the image:
```sh
$ kubectl get pod
NAME                                  READY     STATUS             RESTARTS   AGE
memcached-operator-6b5dc697fb-t62cv   0/1       ImagePullBackOff   0          2m
```
**NOTE:** For testing purposes you can fix this error by making the image public and setting up your cluster to use an insecure image registry. To do so with docker, run the daemon with `--insecure-registry 172.30.0.0/16`.

Reply:
I like to see the error you were pointing out immediately after you mentioned it. The NOTE is a reminder to me on how I should fix the above error.

Reply:
done 👍